src/main/java/jenkinsci/plugins/influxdb/models/InfluxDBClientWrapper.java

```diff
@@ -0,0 +1,305 @@
+            logger.warning("Connection failed: " + e.getMessage());
+        } finally {
+            if (!success) {
+                this.closeAndResetAllClients();
+            }
+        }
+        return success;
+    }
+
+    private boolean tryConnectV2(
+            String url,
+            String organization,
+            String bucket,
+            @Nullable StandardUsernamePasswordCredentials basicAuthCredentials,
+            @Nullable StringCredentials tokenCredentials,
+            boolean usingJenkinsProxy
+    ) {
+        logger.fine("Attempting connection to InfluxDB v2.X API at " + url);
+        boolean success = false;
+        try {
+            InfluxDBClientOptions.Builder options = InfluxDBClientOptions.builder()
+                    .url(url)
+                    .org(organization)
+                    .bucket(bucket)
+                    .okHttpClient(createHttpClient(new URL(url), usingJenkinsProxy));
+            if (basicAuthCredentials != null) {
+                logger.fine("Attempting username/password authentication");
+                options.authenticate(basicAuthCredentials.getUsername(), basicAuthCredentials.getPassword().getPlainText().toCharArray());
+            } else {
+                logger.fine("Attempting token authentication");
+                options.authenticateToken(tokenCredentials.getSecret().getPlainText().toCharArray());
+            }
+            this.v1v2client = InfluxDBClientFactory.create(options.build());
+            if (this.v1v2client.ping() && this.getAPIVersion().startsWith("v2")) {
+                logger.fine("Connection success");
+                success = true;
+            } else {
+                this.v1v2client.close();
+                logger.fine("Connection failed");
+            }
+        } catch (InfluxException | AccessDeniedException | MalformedURLException e) {
+            logger.warning("Connection failed: " + e.getMessage());
+        } finally {
+            if (!success) {
+                this.closeAndResetAllClients();
+            }
+        }
+        return success;
+    }
+
+    private boolean tryConnectV1(String url, String database, String retentionPolicy, @Nullable StandardUsernamePasswordCredentials credentials) {
+        logger.fine("Attempting connection to InfluxDB v1.X API at " + url);
+        boolean success = false;
+        try {
+            if (credentials != null) {
+                logger.fine("Attempting username/password authentication");
+                this.v1v2client = InfluxDBClientFactory.createV1(url, credentials.getUsername(), credentials.getPassword().getPlainText().toCharArray(), database, retentionPolicy);
+            } else {
+                logger.fine("Attempting connection without credentials");
+                this.v1v2client = InfluxDBClientFactory.createV1(url, "", "".toCharArray(), database, retentionPolicy);
+            }
+            if (this.v1v2client.ping() && this.getAPIVersion().startsWith("v1")) {
```

Comment:
After some debugging I found that this is the root cause why the connection test failed. The v1 API version is `1.x.y` instead of `v1.x.y`. By taking the `v` out my local 1.8.10 started to work.

```suggestion
            if (this.v1v2client.ping() && this.getAPIVersion().startsWith("1")) {
```

Reply:
Fixed in 0590bad

Reply:
Fun fact: InfluxDB v1.11 returns `v1.X` instead of `1.X` 🫠